modules/creating-config-files-cluster-install-oci.adoc

```diff
@@ -137,22 +137,22 @@ bootArtifactsBaseURL: <server_URL> <4>
+* For a disconnected network:  After you run the command to generate a minimal ISO Image, the Agent-based installer saves the rootfs image into the `./<installation_directory>/boot-artifacts` directory on your local system. Use your preferred web server, such as any Hypertext Transfer Protocol daemon (`httpd`), to upload rootfs to the location stated in the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file.
 +
-For example, if the `bootArtifactsBaseURL` parameter states \http://192.168.122.20, you would upload the generated `rootfs` image to this location, so that the Agent-based installer can access the image from \http://192.168.122.20/agent.x86_64-rootfs.img. After the Agent-based installer boots the minimal ISO for the external platform, the Agent-based Installer downloads the `rootfs` image from the \http://192.168.122.20/agent.x86_64-rootfs.img location into the system memory.
+For example, if the `bootArtifactsBaseURL` parameter states \http://192.168.122.20, you would upload the generated rootfs image to this location, so that the Agent-based installer can access the image from \http://192.168.122.20/agent.x86_64-rootfs.img. After the Agent-based installer boots the minimal ISO for the external platform, the Agent-based Installer downloads the rootfs image from the \http://192.168.122.20/agent.x86_64-rootfs.img location into the system memory.
 +
 [NOTE]
 ====
-The Agent-based Installer also adds the value of the `bootArtifactsBaseURL` to the minimal ISO Image’s configuration, so that when the Operator boots a cluster’s node, the Agent-based Installer  downloads the `rootfs` image into system memory.
+The Agent-based Installer also adds the value of the `bootArtifactsBaseURL` to the minimal ISO Image’s configuration, so that when the Operator boots a cluster’s node, the Agent-based Installer  downloads the rootfs image into system memory.
 ====
 +
-* For a connected network: You do not need to specify the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file, because the Agent-based Installer, by default, reads the a `rootfs` URL location from \https://rhcos.mirror.openshift.com. After the Agent-based Installer  boots the minimal ISO for the external platform, the Agent-based Installer then downloads the `rootfs` file into your system’s memory from the default {op-system} URL.
+* For a connected network: You do not need to specify the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file, because the Agent-based Installer, by default, reads the a rootfs URL location from \https://rhcos.mirror.openshift.com. After the Agent-based Installer  boots the minimal ISO for the external platform, the Agent-based Installer then downloads the rootfs file into your system’s memory from the default {op-system} URL.
```

Comment:
```suggestion
* For a connected network: You do not need to specify the `bootArtifactsBaseURL` parameter in the `agent-config.yaml` configuration file. The default behavior of the Agent-based Installer is to read the rootfs URL location from \https://rhcos.mirror.openshift.com. After the Agent-based Installer boots the minimal ISO for the external platform, the Agent-based Installer then downloads the rootfs file into your system’s memory from the default {op-system} URL.
```

Reply:
Another extra space and a minimalism suggestion to break up a complex sentence.

Reply:
:heavy_check_mark: